static/js/latest-news.js

```diff
@@ -100,9 +100,9 @@ const createReqListenerForDomContainer = (
   }
 
   try {
-    const latestPinned = latest[0];
+    const latestPinned = data.latest_pinned_articles[0][0];
```

Comment:
Is this array in array or a typo?

Reply:
Its actually nested like this. 
You can see the response here https://jp-ubuntu-com-canonical-web-and-design-pr-132.run.demo.haus/blog/latest-news :)

That is the reason I have wrapped this in a try/catch

Reply:
OK. Bit weird and confusing structure of data, but I guess it works.